src/bindings/python/tests/__init__.py

```diff
@@ -83,7 +83,6 @@ def xfail_test(reason="Mark the test as expected to fail", strict=True):
 xfail_issue_48052 = xfail_test(reason="Dropout op is not supported in traning mode")
 xfail_issue_45180 = xfail_test(reason="RuntimeError: Unsupported dynamic op: ReduceSum")
 xfail_issue_44851 = xfail_test(reason="Expected: Unsupported dynamic op: Broadcast")
-xfail_issue_44854 = xfail_test(reason="Expected: Unsupported dynamic op: VariadicSplit")
```

Comment:
Do I understand correctly that these tests worked thanks to the dynamic shapes support for the VariadicSplit node inside the CPU plugin?

Reply:
This test cannot pass even after CPU plugin has enabled split dynamic shape support, because original validate_and_infer_types() didn't handle the corner case of unknown split_lengths very well and it simple returns dynamic rank pshape and breaks CPU plugin's checks.

Thanks to you, after we handled this corner case and return static rank pshape with only 1 dynamic dimension, CPU can successfully run this test.